configs/sst_cs_stacks-imap-server.yaml

```diff
@@ -6,8 +6,8 @@ data:
   maintainer: sst_cs_infra_services
   packages:
     - cyrus-imapd
-    - procmail
     - xapian-core
+    - imaptest
```

Comment:
This would be a net addition, as the previous config which listed it was labeled `eln-test` (which is invalid).  FWIW imaptest has not been shipped in any previous RHEL versions.  Do you actually plan on adding it?